src/shared/Microsoft.Git.CredentialManager.Tests/Interop/LibGit2Tests.cs

```diff
@@ -123,7 +123,7 @@ public void LibGit2Configuration_TryGetValue_SectionScopeProperty_NullScope_Retu
             var git = new LibGit2();
             using (var config = git.GetConfiguration(repoPath))
             {
-                bool result = config.TryGetValue("user", null, "name", out string value);
+                bool result = config.TryGetValue("user", (string)null, "name", out string value);
```

Comment:
Curious why we need (string) before the null

Reply:
We don't (any more)!

![image](https://user-images.githubusercontent.com/5658207/61298505-3f660000-a7d6-11e9-8cdd-e3d34fd0ce67.png)

I used to have an overload which wasn't a `string` which I removed now.